generals/envs/generalsio_client.py

```diff
@@ -0,0 +1,47 @@
+from socketio import SimpleClient
+
+from generals.agents.agent import Agent
+
+
+class GeneralsBotError(Exception):
+    """Base generals-bot exception
+    TODO: find a place for exceptions
```

Comment:
this is fine here, if the implementation gets too big I can see moving all the code from `/envs/` to elsewhere and creating `exceptions.py`